ooniprobe/View/Settings/SettingsCategoriesTableViewController.m

```diff
@@ -46,11 +48,38 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
     if ([current isEqualToString:@"about_ooni"]){
         [self performSegueWithIdentifier:current sender:self];
     }
+    else if ([current isEqualToString:@"send_email"]){
+        [self sendEmail];
+    }
     else
         [self performSegueWithIdentifier:@"toSettings" sender:self];
     [tableView deselectRowAtIndexPath:indexPath animated:YES];
 }
 
+-(void)sendEmail{
+    if ([MFMailComposeViewController canSendMail]){
+        MFMailComposeViewController *mailer = [[MFMailComposeViewController alloc] init];
+        mailer.mailComposeDelegate = self;
+        [mailer setSubject:[NSString stringWithFormat:@"[bug-report] OONI Probe iOS %@", [[[NSBundle mainBundle] infoDictionary] valueForKey:@"CFBundleShortVersionString"]]];
+        NSArray *toRecipients = [NSArray arrayWithObjects:@"contact@openobservatory.org", nil];
+        [mailer setToRecipients:toRecipients];
+        NSString *device = [NSString stringWithFormat:@"</br></br></br>app_version %@</br>device_model %@</br>os_version %@", [[[NSBundle mainBundle] infoDictionary] valueForKey:@"CFBundleShortVersionString"], [UIDevice currentDevice].modelName, [UIDevice currentDevice].systemVersion];
+        [mailer setMessageBody:device isHTML:YES];
```

Comment:
Can we make the email that users send not HTML by default? I think this will make it easier for us to process them in the future if we want to.

Reply:
ok

Reply:
Updated